full-moon/src/ast/parser_structs.rs

```diff
@@ -315,6 +317,14 @@ impl AstResult {
                             continue;
                         }
 
+                        if block_has_last_stmt {
+                            use crate::node::Node;
```

Comment:
Let's just put this in the beginning

Reply:
I was lazy because it clashes with start_position / end_position on TokenReference. If we stick it at the top we need to disambiguate all other usages

Reply:
Done in last commit, wasn't that bad. Can revert if we want to